mint-auditor/src/db/models/block_balance.rs

```diff
@@ -1,42 +1,54 @@
 // Copyright (c) 2018-2022 The MobileCoin Foundation
 
-pub use super::models::BlockBalance;
-
-use super::{schema, transaction, Conn};
-use crate::Error;
+use crate::db::{schema::block_balance, transaction, Conn, Error};
```

Comment:
Optional/Equivalent: Use `super` instead of a qualified mod path.

```suggestion
use super::{schema::block_balance, transaction, Conn, Error};
```